src/examples/java/com/nylas/examples/other/EventsExample.java

```diff
@@ -95,6 +97,11 @@ protected static void basicEventCrud(Events events, Calendar primary) throws IOE
 		created.setLocation("Lake Merritt");
 		created.setParticipants(Arrays.asList(partier, partier1, partier2, partier3));
 		created.setRecurrence(new Recurrence(startTz.getId(), Arrays.asList("RRULE:FREQ=WEEKLY;BYDAY=TH")));
+
+		Map<String, String> metadata = new HashMap<>();
+		metadata.put("event_category", "gathering");
+		created.setMetadata(metadata);
```

Comment:
alternatively, `created.setMetadata(Maps.of("event_category", "gathering"));`
(using `com.nylas.Maps`)
but this is fine too.

Reply:
For now we will keep it this way as `com.nylas.Maps` is a private class.